src/pages/Checkout/CheckoutPage.vue

```diff
@@ -547,11 +545,13 @@ export default {
 			const transactionData = {
 				transactionId: numeral(transactionId).value(),
 				itemTotal: this.totals.itemTotal,
-				loans: _map(this.loans, loan => {
-					return _pick(loan, ['__typename', 'id', 'price']);
+				loans: this.loans.map(loan => {
+					const { __typename, id, price } = loan;
+					return { __typename, id, price };
```

Comment:
did you see any difference in the return value if there are no loans present?

Reply:
These will both return an empty array